CSS/main.css

```diff
@@ -1,7 +1,23 @@
-.heading__title {
-    font-family: 'Noto', serif;
+.content__title {
+    font-weight: 700;
+    font-size: 36px;
 }
 
-.content__specifications {
-    font-family: 'Nunito';
-}
\ No newline at end of file
+.content__subtitle {
+    font-size: 24px;
+    font-weight: 400;
+}
+
+.copy__specifications {
+    font-size: 16px;
+    font-weight: 400;
+    margin-bottom: 80px;
+    margin-top: 50px;
+}
+
+.btn__specifications {
+    display: flex;
+    font-size: 16px;
```

Comment:
font-size property is already 16px since it is the browser default, please remove this line